doc/dev/lib.xml

```diff
@@ -12,6 +12,61 @@ kernel among the developers.
 
 <Chapter Label="Chap-Lib">
 <Heading>Maintaining the &GAP; kernel and library</Heading>
+
+<Section Label="Sect-debugging">
+<Heading>Debugging the GAP Kernel</Heading>
+
+The GAP kernel supports a variety of options which enabled when running <C>configure</C>,
```

Comment:
`GAP` should be `&GAP;` throughout because as an entity it is displayed in a different font (as a `<Package` element).

Reply:
Required: This sentence seems to be missing a word, like "which *can* be enabled *by* running" ? Or maybe reorder the sentence for readability (and use the appropriate XML for filenames):

> The GAP kernel supports a variety of compile-time options to help writing and debugging GAP kernel code.
> They can be enabled by passing one or more of the following options to the <F>configure</F> script.